Uniswap/interface
# chore: Replace widget skeleton with internal swap skeleton

## Description
Removes the dependency for rendering a Swap skeleton imported from @uniswap/widgets 

Received some new designs from @infredible - https://www.figma.com/file/kNSDBMpOzxSTOP6MerohLm/Web-Design-Spec?type=design&node-id=13335-127826&t=CAu5FD8kCowy3z7c-0 

I haven't finished the whole thing ^ but close enough equivalent for now. Ticket for finishing it: WEB-2030



| Before       | After (Desktop) | After (Mobile) |
| ------------ |---------------- | -------------- |
|<img width="400" alt="image" src="https://github.com/Uniswap/interface/assets/59578595/3fc3b3d0-9ecc-4743-b6f0-dc6a5124f300">|<img width="406" alt="image" src="https://github.com/Uniswap/interface/assets/59578595/b67fed19-8b2e-4b2b-8a0d-8ebbde220e48">| not shown on mobile  |

Note I updated the Swap arrow to be the same one as in the Swap component


### QA (ie manual testing)
If you want to see this on Vercel, I recommend going to the token details page, then going to Network > Throttling > No network > REFRESH to see the loading state show

### Automated testing
Added snapshot test
